framework_tvb/tvb/core/adapters/abcadapter.py

```diff
@@ -523,13 +524,39 @@ def build_adapter(stored_adapter):
             LOGGER.exception(msg)
             raise IntrospectionException(msg)
 
-    def review_operation_inputs(self, parameters):
-        # TODO: implement this for neoforms
+    def review_operation_inputs(self, operation):
         """
         :returns: a list with the inputs from the parameters list that are instances of DataType,\
             and a dictionary with all parameters which are different than the declared defauts
         """
-        return {}, None
+        changed_attr = {}
+        inputs_datatypes = []
+        view_model = self.load_view_model(self, operation)
+        form_model = self.get_view_model_class()()
+        form_fields = self.get_form_class()().fields
+
+        for field in form_fields:
+            if not isinstance(field, TraitDataTypeSelectField):
+                attr_form = getattr(form_model, field.name)
```

Comment:
I would call this attr_default, to be more clear what we are comparing here.
This is ok for handling analyzers/creators, but I don't think it is enough to support also simulations.